sagemaker-training-compiler/huggingface/tensorflow_tune_batch_size/finding_max_batch_size_for_model_training.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>Use the paper name for link text: <code>Accurate, Large Minibatch SGD: Training ImageNet in 1 Hour </code>	. Also why are you linking that particular paper? How does it relate to SM Training Compiler?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Added more explanation and fixed link text